bcos-executor/src/executive/StorageWrapper.h

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include "../Common.h"
+#include "bcos-framework/storage/StorageInterface.h"
+#include "bcos-framework/storage/Table.h"
+#include "bcos-table/src/StateStorage.h"
+#include <boost/iterator/iterator_categories.hpp>
+#include <boost/throw_exception.hpp>
+#include <optional>
+#include <thread>
+#include <vector>
+
+namespace bcos::executor
+{
+using GetPrimaryKeysReponse = std::tuple<Error::UniquePtr, std::vector<std::string>>;
+using GetRowResponse = std::tuple<Error::UniquePtr, std::optional<storage::Entry>>;
+using GetRowsResponse = std::tuple<Error::UniquePtr, std::vector<std::optional<storage::Entry>>>;
+using SetRowResponse = std::tuple<Error::UniquePtr>;
+using OpenTableResponse = std::tuple<Error::UniquePtr, std::optional<storage::Table>>;
+
+
+class StorageWrapper
```

Comment:
Abstracting StorageWrapper seems to be unnecessary, SyncStorageWrapper class doing as well if you add other member fields.